aten/src/ATen/core/function_schema.h

```diff
@@ -59,14 +58,18 @@ struct Argument {
     return is_inferred_type_;
   }
 
+  static std::string formatInferredTypeHint(const std::string& name) {
+    return c10::str(
+        "Inferred '",
+        name,
+        "' to be of type 'Tensor' ",
+        "because it was not annotated with an explicit type.\n");
+  }
```

Comment:
I made a mistake, can you move this back? The error message is slightly different for an inferred arg vs. an inferred *value* so we cannot refactor as easily as I thought. So sorry about this, it's my fault.

Reply:
OK sure